Installation Guide: Setup files (server and proxy) (#676)
SUSE/doc-susemanager: modules/installation/pages/proxy-registration.adoc

image::proxy-bootstrap.png[]

. Create the SUSE Manager Tools Repository for bootstrapping, see xref:client-configuration:creating-a-tools-repository.adoc[Create Tools Repository].
. Bootstrap the client with the bootstrap script.
For more information, see xref:client-configuration:registration-bootstrap.adoc[].

> . In case of a Salt client, accept the key on the menu:Salt[Keys] page by clicking the check mark.

Reviewer: ```suggestion
. For Salt clients, accept the key on the menu:Salt[Keys] page by checking the appropriate checkbox.
```

When accepted, it will appear in the menu:Systems[Overview].

> . Check via menu:System Details[Software > Software Channels] that the four proxy channels ([systemitem]``Pool`` and [systemitem]``Updates`` for [systemitem]``SLE-PRODUCT`` and [systemitem]``SLE-MODULE``) plus the recommended channels are selected.

Reviewer: ```suggestion
. Navigate to menu:System Details[Software > Software Channels], and check that the four proxy channels ([systemitem]``Pool`` and [systemitem]``Updates`` for [systemitem]``SLE-PRODUCT`` and [systemitem]``SLE-MODULE``) plus the recommended channels are selected.
```
